src/jdk.httpserver/share/classes/sun/net/httpserver/ContextList.java

```diff
@@ -26,20 +26,25 @@
 package sun.net.httpserver;
 
 import java.util.*;
-import com.sun.net.httpserver.*;
-import com.sun.net.httpserver.spi.*;
 
 class ContextList {
 
     final static int MAX_CONTEXTS = 50;
```

Comment:
@Michael-Mc-Mahon `MAX_CONTEXTS` is not used anywhere, can it be removed?

Reply:
It can.